data/schema.graphql

```diff
@@ -12311,6 +12311,9 @@ type SaleArtwork implements ArtworkEdgeInterface & Node {
 
   # Singular estimate field, if specified
   estimateCents: Int
+
+  # A formatted description of when the lot starts or ends or if it has ended
+  formattedStartDateTime: String
```

Comment:
A rebase might get rid of this diff since the schema has already been updated in `main`